.github/ISSUE_TEMPLATE/component-request-new-component.md

```diff
@@ -9,19 +9,24 @@ labels: component/new, status/triage, enhancement
 Thank you for taking your time to request a new component for the Synergy Design System. Please make sure to answer the questions below to give us more information about your request.
 -->
 
-**Version**:
+**Questions**
 
-I am using the following version of the component library: _____
+- [ ] I have looked at the documentation site and was not able to find the component. <!-- link zur seite??? -->
+- [ ] I have looked into the Figma boards and was not able to find the component. <!-- kommt jeder auf figma??? -->
```

Comment:
There will be a public Figma board later (I assume). A link will also be part of the documentation. Please remove comment before pushing ;)